docs/src/integrations/retrying-transactions.md

```diff
@@ -0,0 +1,183 @@
+
+![Dropped due to Minority Fork (Before Processed)](/img/rt-dropped-minority-fork-pre-process.png)
+
+### After a transaction is processed and before it is finalized
+
+In the event a transaction references a `recentBlockhash` from a minority fork, it’s still possible for the transaction to be processed. In this case, however, it would be processed by the leader on the minority fork. When this leader attempts to share its processed transactions with the rest of the network, it would fail to reach consensus with the majority of validators that do not recognize the minority fork. At this time, the transaction would be dropped before it could be finalized.
+
+![Dropped due to Minority Fork (After Processed)](/img/rt-dropped-minority-fork-post-process.png)
+
+## Handling Dropped Transactions
+
+While RPC nodes will attempt to rebroadcast transactions, the algorithm they employ is generic and often ill-suited for the needs of specific applications. To prepare for times of network congestion, application developers should customize their own rebroadcasting logic.
+
+### An In-Depth Look at sendTransaction
+
+When it comes to submitting transactions, the `sendTransaction` RPC method is the primary tool available to developers. `sendTransaction` is only responsible for relaying a transaction from a client to an RPC node. If the node receives the transaction, `sendTransaction` will return the transaction id that can be used to track the transaction. A successful response does not indicate whether the transaction will be processed or finalized by the cluster.
+
+```
+Request Parameters
+- `transaction`: `string` - fully-signed Transaction, as encoded string
+- (optional) `configuration object`: `object`
+    - `skipPreflight`: `boolean` - if true, skip the preflight transaction checks (default: false)
+    - (optional) `preflightCommitment`: `string` - [Commitment](https://docs.solana.com/developing/clients/jsonrpc-api#configuring-state-commitment) level to use for preflight simulations against the bank slot (default: "finalized").
+    - (optional) `encoding`: `string` - Encoding used for the transaction data. Either "base58" (slow), or "base64". (default: "base58").
+    - (optional) `maxRetries`: `usize` - Maximum number of times for the RPC node to retry sending the transaction to the leader. If this parameter is not provided, the RPC node will retry the transaction until it is finalized or until the blockhash expires.
+
+Response
+- `transaction id`: `string` - First transaction signature embedded in the transaction, as base-58 encoded string. This transaction id can be used with [getSignatureStatuses](https://docs.solana.com/developing/clients/jsonrpc-api#getsignaturestatuses) to poll for status updates.
+```
```

Comment:
can we format this block with markdown instead of the code block? it's hard to read like this

Reply:
![Screenshot 2022-08-01 at 19 44 24](https://user-images.githubusercontent.com/14809642/182232300-0355ed7b-89c3-460c-93f7-19317ae044e1.png)

agreed, these were originally `:::tip` blocks that aren't used anywhere else on the docs site.

Could put them back in and tweak the styling to have similar colors to the code blocks?